### PR TITLE
Add angular-simple-logger@0.1.7 w/ git auto-update

### LINKED ISSUE #10035: [Request] angular-simple-logger
**Library name: angular-simple-logger
**Git repository url: https://github.com/nmccready/angular-simple-logger 
**License(s): MIT
**Official homepage: http://nmccready.github.io/angular-simple-logger/example/

### PR DESCRIPTION
**Pull request by clairetsai818**
close #10035, cc @nmccready

Pull request for issue: #10035 

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/nmccready/angular-simple-logger
 * Official website (optional, not the repository): http://nmccready.github.io/angular-simple-logger/example/
 * License and its reference: MIT
 * GitHub / Bitbucket popularity (required):
   - Count of watchers: 1
   - Count of stars: 35
   - Count of forks: 98

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Forks] on [GitHub]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [x] Has valid tags for each versions (for git auto-update)
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

**Comment by clairetsai818** [comment]
@PeterDaveHello please help me review this, thanks.

File: ajax/libs/angular-simple-logger/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+  "author": "Nicholas McCready",
+  "license": "MIT",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/nmccready/angular-simple-logger.git",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "angular-simple-logger.*"
```
Comment (PeterDaveHello) [inline ajax/libs/angular-simple-logger/package.json]: Do you know that what are the other files for and why to exclude them?

Comment (PeterDaveHello) [inline ajax/libs/angular-simple-logger/package.json]: I prefer to add `**/` prefix under `dist` folder.

**Comment by clairetsai818** [comment]
@PeterDaveHello I've revised the file map. Please help me review this, thanks!

**Comment by PeterDaveHello** [comment]
So ...
> Do you know that what are the other files for and why to exclude them?

**Comment by clairetsai818** [comment]
@PeterDaveHello It looks like a lightweight version of the source file.

**Comment by PeterDaveHello** [comment]
@clairetsai818 so why don't we include them?

**Comment by clairetsai818** [comment]
@PeterDaveHello I fixed the file map. Please help me review this, thanks!

**Comment by PeterDaveHello** [comment]
The PR title and commit message ... `w/` not `/w`

**Pull request merged**